app/javascript/components/mentoring/session/FileViewer.tsx

```diff
@@ -20,9 +20,8 @@ export const FileViewer = ({
         data-highlight-line-numbers={true}
         data-highlight-line-number-start={1}
         style={{ tabSize: indentSize }}
-      >
-        {file.content}
-      </code>
+        dangerouslySetInnerHTML={{ __html: file.content }}
```

Comment:
Do we need to do this? `file.content` is user-submitted and not sanitized, which means we'd open ourselves up to XSS attacks.